docs/CommandHandler.puml

```diff
@@ -0,0 +1,50 @@
+@startuml
+'https://plantuml.com/class-diagram
+skinparam classAttributeIconSize 0
+hide circle
+
+class CommandHandler {
+-{static} logger: Logger
++CommandHandler(): void
++help(): void
++add(input: String, records: Records): void
++list(records: Records): void
++edit(input: String, records: Records): void
++editVisit(input: String, records: Records): void
++delete(nric: String, records: Records): void
++find(input: String, records: Records): void
++exit(input: String): void
++addVisit(input: String, records: Records): void
++Appointment(input: String, appointmentRecord: AppointmentRecord): void
++deleteAppointment(input: String, appointmentRecord: AppointmentRecord): void
++listAppointment(appointmentRecord: AppointmentRecord): void
++findAppointment(input: String, appointmentRecord: AppointmentRecord): void
++deleteAppointment(appointmentRecord: AppointmentRecord): void
+-findNextFieldStart(input: String, currentIndex: int): int
```

Comment:
I like that it is clear which component is responsible for what. Wouldn't all of these methods be a bit overwhelming for a reader? Isn't it too detailed?